keras_nlp/layers/preprocessing/masked_lm_mask_generator_test.py

```diff
@@ -231,27 +166,9 @@ def test_config(self):
 
         # Test cloned masked_lm_masker can be run.
         cloned_masked_lm_masker = MaskedLMMaskGenerator.from_config(config)
-        inputs = tf.ragged.constant(
-            [[0, 1, 2], [0, 1, 2, 3, 4, 5], [0, 1, 2, 3, 4]]
-        )
+        inputs = [[5, 3, 2], [1, 2, 3, 4]]
         cloned_masked_lm_masker(inputs)
 
-    def test_graph_mode_execution(self):
```

Comment:
Does this mean that this no longer works in tf.data?

Reply:
Still works! The test is right below. Just seemed redundant to test `tf.data` and an explicit `tf.function`, so removed the latter.